systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -256,6 +257,7 @@ void testKafkaExporterDifferentSetting() throws InterruptedException, ExecutionE
     }
 
     @ParallelTest
+    @Disabled("Because of this PR https://github.com/strimzi/strimzi-kafka-operator/pull/5137.")
```

Comment:
It's not working even for bundle install type with cluster-wide operator?

Reply:
discussed offline. Make sense now.

Reply:
So, what does it mean? Can you explain it here as well?

Reply:
It works with the cluster-wide operator but I let the implementation of the test for explicit 2 namespaces (which currently doesn't work but with Standa's PR it will work). That's it.